r-package/grf/vignettes/rate.Rmd

```diff
@@ -0,0 +1,216 @@
+
+## Treatment prioritization rules
+We are in the familiar experimental setting (or unconfounded observational study) and are interesting in the problem of determining which individuals to assign a binary treatment $W=\{0, 1\}$ and the associated value of adhering to this treatment assignment. Given some subject characteristics $X_i$ we have access to a subject-specific *prioritization rule* $S(X_i)$ which assigns scores to subjects. This prioritization rule should give a high score to units which we believe to have a large benefit of treatment and a low score to units with a low benefit of treatment. By benefit of treatment, we mean the difference in outcomes $Y$ from receiving the treatment given some subject characteristics $X_i$, as given by the conditional average treatment effect (CATE)
+
+$$\tau(X_i) = E[Y_i(1) - Y_i(0) \,|\, X_i = x],$$
+where $Y(1)$ and $Y(0)$ are potential outcomes corresponding to the two treatment states.
+
+You might ask: why the general focus on an arbitrary rule $S(X_i)$ when you define benefit as measured by $\tau(X_i)$? Isn't it obvious that the estimated CATEs would serve the best purpose for treatment targeting? The answer is that this general problem formulation is quite convenient, as in some settings we may lack sufficient data to power an accurate CATE estimator and have to rely on other approaches to target treatment. Examples of other approaches are heuristics derived by domain experts or simpler models predicting risk scores (where risk is defined as $P[Y = 1 | X_i =  x]$ and $Y \in \{0, 1\}$ with $Y=1$ being an adverse outcome), which is quite common in clinical applications. Consequently, in finite samples we may sometimes do better by relying on simpler rules which are correlated with the CATEs, than on noisy and complicated CATE estimates (remember: CATE estimation is a hard statistical task, and by focusing on a general rule we may circumvent some of the problems of obtaining accurate non-parametric point estimates by instead asking for estimates that *rank* units according to treatment benefit). Also, even if you have an accurate CATE estimator, there may be many to choose from (neural nets/random forests/various metalearners/etc). The question is: given a set of treatment assignment rules $S(X_i)$, which one (if any) should we use?
+
+The tool we propose to employ in this situation is the "RATE". It takes as input a prioritization rule $S(X_i)$ which may be based on estimated CATEs, risk scores, etc, and outputs a scalar metric that can be used to assess how well it targets treatment (it measures the *value* of a prioritization rule while being agnostic to exactly how it was derived). The RATE has two components: the "TOC" and the area under the TOC, which we explain in the following section.
```

Comment:
"is a RATE metric" instead of "is the ''RATE''" and similar in the end "A RATE [metric]"